src/main/java/com/alvarium/contracts/ServiceInfo.java

```diff
@@ -12,7 +12,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  *******************************************************************************/
-package com.alvarium.streams;
+package com.alvarium.contracts;
```

Comment:
Help me understand why this was moved into contracts? This package should only be for types designated for use outside of the SDK whereas it appears ServiceInfo is still only used internally by the SDK.

Reply:
This was an oversight on my side, moved it to utils in fed5b82.